database/db.init

```diff
@@ -0,0 +1,129 @@
+drop schema public cascade;
+create schema public;
+--DROP TABLE public."user";
+
+CREATE TABLE public."user"
+(
+    id       serial,
+    name     character varying(250),
+    email    character varying(250),
+    password character varying(250),
+    CONSTRAINT id PRIMARY KEY (id)
+) WITH (
+      OIDS = FALSE
+    );
+
+CREATE TABLE SUBJECT
+(
+    id          SERIAL PRIMARY KEY UNIQUE,
+    name        VARCHAR,
+    description VARCHAR,
+    user_id     SERIAL REFERENCES public."user" (id)
+);
+
+CREATE TABLE EXAM
+(
+    id          SERIAL PRIMARY KEY UNIQUE,
+    name        VARCHAR,
+    description VARCHAR,
+    subject_id  SERIAL REFERENCES SUBJECT (id),
+    pass_score  VARCHAR,
+    file_name   VARCHAR,
+    date        DATE,
+    user_id     SERIAL REFERENCES public."user" (id),
+    owners      VARCHAR

```

Comment:
How do we want to assign people who aren't the owner but are capable of modifying exams (e.g. by adding some students' exams)?

Reply:
@MateuszKepczynski please have a look